components/ILIAS/Authentication/classes/class.ilSession.php

```diff
@@ -353,58 +334,28 @@ public static function _duplicate(string $a_session_id): string
      *
      * Returns the expiration timestamp in seconds
      *
-     * @param	boolean	If passed, the value for fixed session is returned
      * @return	integer	The expiration timestamp in seconds
      * @static
      *
      */
-    public static function getExpireValue(bool $fixedMode = false): int
+    public static function getExpireValue(): int
     {
-        global $DIC;
-
-        if ($fixedMode) {
-            // fixed session
-            return time() + self::getIdleValue($fixedMode);
-        }
-
-        /** @var ilSetting $ilSetting */
-        $ilSetting = $DIC['ilSetting'];
-        if ($ilSetting->get('session_handling_type', (string) self::SESSION_HANDLING_FIXED) === (string) self::SESSION_HANDLING_FIXED) {
-            return time() + self::getIdleValue($fixedMode);
-        }
-
-        if ($ilSetting->get('session_handling_type', (string) self::SESSION_HANDLING_FIXED) === (string) self::SESSION_HANDLING_LOAD_DEPENDENT) {
-            // load dependent session settings
-            $max_idle = (int) ($ilSetting->get('session_max_idle') ?? ilSessionControl::DEFAULT_MAX_IDLE);
-            return time() + $max_idle * 60;
-        }
-        return time() + ilSessionControl::DEFAULT_MAX_IDLE * 60;
+            return time() + self::getIdleValue();
     }
 
     /**
      *
      * Returns the idle time in seconds
      *
-     * @param	boolean	If passed, the value for fixed session is returned
```

Comment:
We should cleanup this comment in general.

Reply:
just remove the @return or the entire comment?